test/stdlib/POSIX.swift

```diff
@@ -15,7 +15,8 @@ chdir(CommandLine.arguments[1])
 
 var POSIXTests = TestSuite("POSIXTests")
 
-let semaphoreName = "TestSem"
+let semaphoreName = "/TestSem"
```

Comment:
why are you making this change? Can you add a comment explaining this if it is important.

Reply:
The reason for this is that POSIX says a name should start with `/`, but allows implementations to define the behavior if the name does not start with `/`. FreeBSD requires the name to start with `/`. I'll add a comment.